Functions/Assertions/Exist.ps1

```diff
@@ -1,6 +1,7 @@
 
 function PesterExist($value) {
-    return (Test-Path $value)
+    $resolvedPath = $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath($value)
```

Comment:
This line is going to break things (because it strips off the provider name from PS paths), and isn't necessary anyway.  Just the call to `Test-Path -LiteralPath` should be sufficient, I think.